include/net/coap.h

```diff
@@ -147,6 +147,11 @@ enum coap_response_code {
 
```

Comment:
Please change the commit subject of `lib: coap: ...` to `net: coap: ....` as we have typically named coap commits this way.